EMIP/3001-4000/EMIP03003.xml

```diff
@@ -28,22 +28,258 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <msIdentifier>
                         <repository ref="INS0447EMIP">UNESCO Collection</repository>
                         <collection>EMIP</collection>
-                        <idno facs="EMIP/Codices/3003/" n="98">Unesco </idno>
+                        <idno facs="EMIP/Codices/3003/" n="98">Unesco</idno>
```

Comment:
I know you did not create the file, but do you happen to know the unesco shelfmark? Then it could be included here as well.

Reply:
It is UNESCO 8-9; I have now created a new PR with a fix https://github.com/BetaMasaheft/Manuscripts/pull/1977 (I did not see this PR)